examples/010_harmonic_example.py

```diff
@@ -0,0 +1,241 @@
+server = connect_to_or_start_server(port=50052)
+composite_files_on_server = get_continuous_fiber_example_files(server, "harmonic")
+
+# %%
+# Create a composite model
+composite_model = CompositeModel(composite_files_on_server, server)
+
+
+# %%
+# Define a failure criterion
+combined_fc = CombinedFailureCriterion(
+    name="My Failure Criteria",
+    failure_criteria=[
+        MaxStressCriterion(),
+        TsaiWuCriterion(),
+    ],
+)
+
+# %%
+# Obtain stresses and strains and compute max IRF over all phases and frequencies
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Get complex stresses and strains at all frequencies
+#
+stress_operator = composite_model.core_model.results.stress.on_all_time_freqs()
+stress_operator.inputs.bool_rotate_to_global(False)
+
+strain_operator = composite_model.core_model.results.elastic_strain.on_all_time_freqs()
+strain_operator.inputs.bool_rotate_to_global(False)
+
+# %%
+# Get operators that provide material data
+#
+material_operators = get_material_operators(
+    composite_model.data_sources.rst,
+    composite_model.data_sources.engineering_data,
+    get_unit_system(composite_model.data_sources.rst),
+)
+
+# %%
+# Define the frequency sweep. This is relatively coarse so the example does not take
+# too much time.
+#
+min_frequency = -180
+max_frequency = 180
+frequency_step = 60
+sweep_phases = range(min_frequency, max_frequency, frequency_step)
+
+# %%
+# Initialize result containers. We create a result container for the failure value,
+# the failure mode and the layer index in which the failure occurs.
+#
+
+# This is a bit confusing, but DPF uses the same label for Frequency and Time
+FREQ_LABEL = "time"
+PHASE_LABEL = "phase"
+FAILURE_LABEL = "failure_label"
+all_phases_and_freqs_failure_value_fc = dpf.FieldsContainer()
+all_phases_and_freqs_failure_value_fc.labels = [FREQ_LABEL, PHASE_LABEL]
+
+all_phases_and_freqs_critical_layer_fc = dpf.FieldsContainer()
+all_phases_and_freqs_critical_layer_fc.labels = [FREQ_LABEL, PHASE_LABEL]
+
+all_phases_and_freqs_failure_mode_fc = dpf.FieldsContainer()
+all_phases_and_freqs_failure_mode_fc.labels = [FREQ_LABEL, PHASE_LABEL]
+
+
+# %%
+# Loop over all the phases and evaluate the failure criterion.
+# The result is added to the field containers declared in the previous cell.
+# In the end each container contains a field for each combination of frequency
+# and phase
+#
+for phase in sweep_phases:
+    # Stress & strains at sweeping phase
+    stress_at_phase = dpf.operators.math.sweeping_phase_fc(
+        fields_container=stress_operator, angle=float(phase), unit_name="deg", abs_value=False
+    )
+
+    strain_at_phase = dpf.operators.math.sweeping_phase_fc(
+        fields_container=strain_operator, angle=float(phase), unit_name="deg", abs_value=False
+    )
+
+    failure_evaluator = dpf.Operator("composite::multiple_failure_criteria_operator")
```

Comment:
Unrelated to this PR, but should we define these operator names as constants somewhere in `pydpf-composites`?

Reply:
Not sure. If a user works with the native operators, then it helps to search in the help.